packages/@react-spectrum/buttongroup/docs/ButtonGroup.mdx

```diff
@@ -0,0 +1,82 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/buttongroup';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Button} from '@react-spectrum/button';
+import {ButtonGroup} from '@react-spectrum/buttongroup';
+import {View} from '@react-spectrum/view';
+```
+
+# Button
+
+<p>{docs.exports.ButtonGroup.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ButtonGroup']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/button/#Button-group-overflow'}
+  ]} />
+
+## Example
+To see the orientation switch, reduce the width of your window.
+```tsx example
+<ButtonGroup width="50%">
+  <Button variant="primary">Rate Now</Button>
+  <Button variant="secondary">No, thanks</Button>
+  <Button variant="secondary">Remind me later</Button>
+</ButtonGroup>
+```
+
```

Comment:
Should there a Content section here? I think it is important to state that ButtonGroup specifically expects Buttons as it's children since it passes the modularized spectrum-ButtonGroup-Button class name via slots (prob don't need to go into that much detail, can leave out the slots bit).